src/components/tabs/__next__/tabs.style.ts

```diff
@@ -76,10 +76,10 @@ export const StyledScrollButton = styled.button<{
   border-radius: 0px;
   border-top-right-radius: 8px;
   border-top-left-radius: 8px;
-  border-color: #8b8b8bff;
+  border-color: var(--tab-border-active-alt);
   background: white;
   border: none;
-  border-bottom: 2px solid #8b8b8bff;
+  border-bottom: 2px solid var(--tab-border-active-alt);
```

Comment:
comment (non-blocking): if `border-color` is being set above do we need to define it here again, maybe we could just define `border-style` and `border-width`